sdk/storage/azure-storage-blob/azure/storage/blob/_download.py

```diff
@@ -189,11 +196,13 @@ def _download_chunk(self, chunk_start, chunk_end):
             content_length = download_range[1] - download_range[0] + 1
             chunk_data = b"\x00" * content_length
         else:
+            md5_validation = self.validate_content is True or self.validate_content == ChecksumAlgorithm.MD5
```

Comment:
nit:
I think
`md5_validation = self.validate_content or self.validate_content == ChecksumAlgorithm.MD5`
is logically equivalent without the `is True`